vars/branchRepo.groovy

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/groovy
+
+def call(String gitRepoUrl, String gitRef, String checkoutDir, String branchName, Closure body = {}) {
```

Comment:
@grdryn Added a default empty body object to both tagRepo and branchRepo methods to make this optional.